packages/component/src/SendBox/TextBox.js

```diff
@@ -115,6 +116,7 @@ const PREVENT_DEFAULT_HANDLER = event => event.preventDefault();
 
 const TextBox = ({ className }) => {
   const [{ sendBoxTextWrap }] = useStyleOptions();
+  const [, setEmoji] = useRenderEmoji();
```

Comment:
For a verb-based hook name, e.g. use*Render*Emoji. This should return a function, when called, to render emoji.

Can you change it to a noun-based hook name?